### DISCUSSION TITLE
Error installing Globaleaks software

### DISCUSSION THREAD

**Discussion by WhistleNoob** [globaleaks/globaleaks-whistleblowing-software#3948] [Q&A]
Dear all, I have a problem, trying to install globaleaks software following the instructions provided here https://docs.globaleaks.org/en/main/setup/InstallationGuide.html 

I had to add the globaleaks repository with this command:

sudo add-apt-repository "deb "https://deb.globaleaks.org jammy/ release"

but when I press enter to add it I get this error:

> Found existing deb entry in /etc/apt/sources.list.d/archive_uri-https_deb_globaleaks_org-jammy.list
> Adding deb entry to /etc/apt/sources.list.d/archive_uri-https_deb_globaleaks_org-jammy.list
> Found existing deb-src entry in /etc/apt/sources.list.d/archive_uri-https_deb_globaleaks_org-jammy.list
> Adding disabled deb-src entry to /etc/apt/sources.list.d/archive_uri-https_deb_globaleaks_org-jammy.list
> E: Malformed entry 1 in list file /etc/apt/sources.list.d/archive_uri-http_deb_globaleaks_org-jammy.list (absolute Suite Component)
> E: Malformed entry 1 in list file /etc/apt/sources.list.d/archive_uri-https_deb_globaleaks_org-jammy.list (absolute Suite Component)
> E: The list of sources could not be read.
> 

so it's impossible to retreive the packages to install globaleaks. 

Is there anyone out there that can help me too understan what to do to solve this problem?

Thank you

**Comment by evilaliv3** [comment]
Thank you.

Please retry with a new system by just running our official script and without adding the repository manually and let me know the output of debug output of the installation script.

**Reply by WhistleNoob** [reply]
Ciao, ho notato che sei italiano e quindi ti scrivo in italiano ^_^ allora, premessa, ho prima testato installazione e tutto creando una macchina virtuale in VMware con ubuntu server lts 22.04 , ho aggiornato, e poi ho immesso i comandi di installazione seguenti:

wget https://deb.globaleaks.org/install-globaleaks.sh
chmod +x install-globaleaks.sh
./install-globaleaks.sh

tutto è andato bene, ho installato, configurato col wizard, fatto una demo interna per l'azienda etc. Dopo di che sono passato al server fisico vero e proprio in cloud. 

Ho installato sempre la stessa versione di linux, ho aggiornato, ho immesos i comandi di installazione globaleaks e...sorpresa! mi compariva il messaggio di errore che indica l'impossibilità di trovare i pacchetti nei repository!

Ho quindi inserito a mano i repository globaleaks ma niente, ad un certo punto è uscito quell'errore che ho indicato nel primo post.

Sono però riuscito a risolvere in questo modo:

ho spostato il file sources.list e ne ho creato uno nuovo immettendo solo i repository ubuntu ufficiali
sono andato nella directory /etc/apt/sources.list.d/ e ho cancellato tutti i files presenti.

a quel punto ho riavviato il server e ho lanciato i comandi di aggiornamento che sono andati a buon fine, quindi ho provato di nuovo ad installare globaleaks e...BAM, il software si è installato correttamente.

Non ho ben capito come mai prima il file sources.list avesse dei problemi, o fosse corrotto, sta di fatto che una volta ripristinato, la piattaforma whistleblowing si è installata correttamente.

Problema risolto direi, grazie per il tuo pronto intervento ^_^
